### DISCUSSION TITLE
Redisplay error messages

### DISCUSSION THREAD

**Discussion by seancallanan** [LazyVim/LazyVim#213] [Q&A] [answered]
I see some messages displaying and they disappear before I can see them, it there a way to reshow the messages.  I tried :messages but it doesn't show anything.  I've also searched for log files but with not success.

BTW: This is an awesome piece of work, thanks you

**Comment by folke** [comment, marked as answer]
You can use `:Noice` to show all messages / notifications.

There's also additional keymaps for more stuff. See https://www.lazyvim.org/keymaps#noicenvim
